network-api/networkapi/wagtailpages/models.py

```diff
@@ -735,8 +735,21 @@ def entries_by_tag(self, request, tag, *args, **kwargs):
     category routes
     """
 
+    # helper function to resolve category slugs to actual objects
```

Comment:
before, this code only existed in the `category/somecategory` route handler, so it never kicked in for the "load more" request (which uses a different route) and so the actual result fetching completely failed.